components/director/internal/domain/formationassignment/notifications.go

```diff
@@ -344,3 +358,29 @@ func convertFormationAssignmentFromModel(formationAssignment *model.FormationAss
 		Value:       config,
 	}
 }
+
+func (fan *formationAssignmentNotificationService) extractCustomerTenantContext(ctx context.Context, internalTenantID string) (*webhook.CustomerTenantContext, error) {
+	tenantObject, err := fan.tenantRepository.Get(ctx, internalTenantID)
+	if err != nil {
+		return nil, err
+	}
+
+	customerID, err := fan.tenantRepository.GetCustomerIDParentRecursively(ctx, internalTenantID)
+	if err != nil {
+		return nil, err
+	}
+
+	var accountID *string
+	var path *string
+	if tenantObject.Type == tenant.Account {
```

Comment:
Is it possible for the tenant to be sth different from Account or ResourceGroup?

Reply:
It shouldn't be, because both Formation/FormationAssignment are scoped to Account/ResourceGroup and we store that Account/ResourceGroup as tenant field in them.

For example, one should not be able to CRUD a Formation from a subaccount.